src/game/game.py

```diff
@@ -0,0 +1,14 @@
+from google.appengine.ext import ndb
+from google.appengine.ext.ndb import polymodel
+
+
+class Game(polymodel.PolyModel):
+    date = ndb.DateTimeProperty(auto_now_add=True)
+    players = ndb.KeyProperty("User", repeated=True)
+    point_data = ndb.StructuredProperty(PointData, repeated=True)
+    winners = ndb.KeyProperty("User", repeated=True)
```

Comment:
To me it would make sense to relay the type in this variable name:
`player_keys`
`winner_keys`